keps/sig-node/runtime-class-scheduling.md

```diff
@@ -203,6 +202,36 @@ will be a clear explanation of why. For example:
 0/10 Nodes are available: 5 nodes do not have enough memory, 5 nodes don't match RuntimeClass
 ```
 
+### RuntimeController
```

Comment:
nit: I would call this `Runtime Admission Controller` to clarify that this is not a controller with a control loop that runs all the time and monitors things in the cluster.

Reply:
I agree

Reply:
Looks like the convention of the admission plugins is to just omit the controller piece altogether - so this would just be "Runtime". I'll clarify the name in the KEP though.

Reply:
or rename to `RuntimeTopology` ?

Reply:
The plan is to reuse the controller for other RuntimeClass-related admission controls. In particular PodOverhead. So I don't think it makes sense to include topology in the name.

Reply:
agree with a single admission plugin for a common feature.

Reply:
@tallclair - have a suggested controller name in mind?  ContainerRuntime? Runtime is pretty overloaded...

Reply:
I went with `RuntimeClass` for the controller name. Pretty self-explanatory.